modules/virt-template-vm-pxe-config.adoc

```diff
@@ -55,8 +55,11 @@ spec:
           image: kubevirt/fedora-cloud-container-disk-demo
       - cloudInitNoCloud:
           userData: |
-            #!/bin/bash
-            echo "fedora" | passwd fedora --stdin
+           #cloud-config
+           user: fedora
+           password: fedora
+           chpasswd: {expire: False}
         name: cloudinitdisk
     status: {}
+...
```

Comment:
is this correct / expected ?

Reply:
According to our style guidelines, we are required to add the ellipsis (...) at the end of the code snippet to indicate that the file continues after the quoted block.